src/TheThingsNetwork.cpp

```diff
@@ -396,19 +402,51 @@ void TheThingsNetwork::autoBaud()
   while (attempts-- && length == 0)
   {
     delay(100);
+    // Send break + Autobaud 
     modemStream->write((byte)0x00);
     modemStream->write(0x55);
     modemStream->write(SEND_MSG);
-    sendCommand(SYS_TABLE, 0, true, false);
-    sendCommand(SYS_TABLE, SYS_GET, true, false);
-    sendCommand(SYS_TABLE, SYS_GET_VER, false, false);
-    modemStream->write(SEND_MSG);
-    length = modemStream->readBytesUntil('\n', buffer, sizeof(buffer));
+    // check we can talk
+    length = checkModuleAvailable();
+    
+    // We succeded talking to the module ?
+    baudDetermined = length > 0 ;
   }
   delay(100);
   clearReadBuffer();
   modemStream->setTimeout(10000);
-  baudDetermined = true;
+}
+
+bool TheThingsNetwork::isSleeping()
+{  
+  return baudDetermined ? false : true ;
```

Comment:
`return !baudDetermined;`

Reply:
Also is this a sensible check? Semantically it's not clear to me.

Reply:
yes it's the same thing,  changed it